index.html

```diff
@@ -43,26 +83,34 @@ <h4 class="h4">Add a new book:</h4>
         </form>
       </section>
 
-      <!------------------->
+      <!----------------------->
       <!-- SECTION BOOK LIST -->
-      <!------------------->
-      <section class="container bg-secondary-subtle p-3">
+      <!----------------------->
+      <section id="booklist" class="container bg-secondary-subtle p-3">
         <h2 class="h4">List of all books</h2>
 
         <!-- Container of all books -->
-        <div class="container container--book">
-          <!--
-          <div class="row border border-success-subtle rounded-2 align-items-center p-2 fs-5 mt-2">
-            <div class="col col-12 col-md-8 mb-3 mb-md-0 text-center text-md-start">
-              <p class="book-title mb-0">
-                <strong>Book</strong> by <strong>Author</strong>
-              </p>
-            </div>
-            <div class="col col-12 col-md-4">
-              <button class="btn btn-danger w-100 fs-5">Remove</button>
-            </div>
-          </div>
-          -->
+        <!-- The content of this section will be added by JavaScript -->
+        <div class="container container--book"></div>
+      </section>
+
+      <!----------------------->
+      <!-- SECTION CONTACCT -->
+      <!----------------------->
+      <section id="contact" class="d-none">
+        <div class="container pt-3 pb-3 bg-secondary-subtle text-center">
+          <h2>Contact Information</h2>
+          <p>
+            Do you have any questions or you just want to say hello? you can
+            reach out to us!
+          </p>
+          <ul class="list-group">
+            <li class="list-group-item"><strong>Our Eamil: </strong> anwarhussaini160@gmail.com</li>
+            <li class="list-group-item"><strong>Our Phone number: </strong> +93 76 459 7266</li>
+            <li class="list-group-item">
+              <strong>Our Address: </strong> Nakamise-dori: 22, 8453 Tokyo, Japane
+            </li>
+          </ul>
         </div>
```

Comment:
- [ ] This footer section is missing from all the three sections, kindly look into it and implement it according to the given template. 👍 

## Wireframe
<img width="708" alt="Untitled" src="https://github.com/M-Anwar-Hussaini/Awesome-Library/assets/103386183/a48b2f7e-9a15-4d5a-b979-a32751b6d9c4">